src/ripple/app/tx/impl/CancelTicket.cpp

```diff
@@ -32,6 +32,9 @@ CancelTicket::preflight (PreflightContext const& ctx)
     if (! ctx.rules.enabled(featureTickets))
         return temDISABLED;
 
+    if (! ctx.rules.enabled(fix1543) && ctx.tx.getFlags() & tfUniversalMask)
```

Comment:
This condition doesn't look right. We want the extra check if `fix1543` is enabled, not disabled.

Reply:
I meant to remove the amendment check entirely, actually. `CreateTicket` and `CancelTicket` aren't enabled yet, so we can modify their semantics directly.

Reply:
Cool, although Escrow and PayChan have the same backward condition and need to be fixed.

Reply:
Yep, done.